Formula/opencascade.rb

```diff
@@ -0,0 +1,68 @@
+class Opencascade < Formula
+  desc "3D modeling and numerical simulation software for CAD/CAM/CAE"
+  homepage "https://www.opencascade.com/content/core-technology"
+  url "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=refs/tags/V7_2_0;sf=tgz"
+  version "7.2.0"
+  sha256 "adb6fc44efebdb09e30a487fa412afae9599992dc771b0039b4ebf306891a558"
+  head "http://git.dev.opencascade.org/gitweb/?p=occt.git;a=snapshot;h=HEAD;sf=tgz"
+
+  option "with-extras", "Install documentation (~17 MB), source files (~113 MB), samples and templates"
```

Comment:
don't think we need this option

Reply:
Opencascade is in essence an SDK or, Kit in Apple parlance.  This means that there are two potential use cases or scenarios in the Homebrew ecosystem to consider. 

*CAD User* - Homebrew users installing a program/package that uses the opencascade SDK libraries
*CAD Developer* - Homebrew users who are developing an application that uses opencascade 

The formula defaults satisfy the *CAD User* however *CAD Developers* require access to the API documentation and source code hence the legacy `--with-extras` developer-convenience option.  

It is clear from the `brew analytics` above that `--with-extras` is rarely used but if we drop `--with-extras`, we will force *CAD Developers* to manually install/configure opencascade (they can use the formula as a reference but it certainly isn't convenient) outside the Homebrew ecosystem.

If everyone agrees that we don't need to support the *CAD Developer* use case, then, at a minimum, we should install the documentation as @fxcoudert suggested at the small of cost of increased bottle size.

I'm not advocating one or the other, I'm simply sharing context for us to make an informed choice/decision.  Please advise and I will make it happen.